docker-app/qfieldcloud/subscription/models.py

```diff
@@ -726,6 +726,9 @@ def create_subscription(
             ), "Creating a trial plan requires `active_since` to be a valid datetime object"
 
             active_until = active_since + timedelta(days=config.TRIAL_PERIOD_DAYS)
+            logging.info(
+                f"Creating trial subscription from {active_since} to {active_until}"
```

Comment:
```suggestion
                f"Creating trial subscription from {active_since=} to {active_until=}"
```

No strong opinion, just I think makes reading logs easier later.